src/pages/my-events.tsx

```diff
@@ -73,7 +73,7 @@ const Table = observer(() => {
                         <TabItem value='reviewed' label='Review'>
```

Comment:
`Tabs` and `TabItems` are components from docusaurus [👉 Tabs](https://docusaurus.io/docs/markdown-features/tabs).

Only the `label` must be translated (the `value` should not be translated, since it is used to display the correct tab).
-->

```tsx
<TabItem value='reviewed' label={translate({message: 'Review', id: 'my-events.tab.review'})}>
```